ibis/expr/window.py

```diff
@@ -420,7 +420,8 @@ def trailing_window(preceding, group_by=None, order_by=None):
     preceding : int, float or expression of intervals, i.e.
         ibis.interval(days=1) + ibis.interval(hours=5)
         Int indicates number of trailing rows to include;
-        0 includes only the current row.
+        0 includes only the current row, 1 includes the current row and one
```

Comment:
need to update for trailing_range_window as well?

Reply:
trailing_range_window is a time range window and not a row-based window so this doesn't apply.